Texassemble/texassemble.cpp

```diff
@@ -1244,10 +1269,16 @@ int __cdecl wmain(_In_ int argc, _In_z_count_(argc) wchar_t* argv[])
                 case CMD_ARRAY:
                 case CMD_CUBEARRAY:
                 case CMD_MERGE:
+                case CMD_CUBE_FROM_HC:
+                case CMD_CUBE_FROM_VC:
+                case CMD_CUBE_FROM_VC_FNZ:
+                case CMD_CUBE_FROM_HT:
+                case CMD_CUBE_FROM_HS:
+                case CMD_CUBE_FROM_VS:
                     break;
 
                 default:
-                    wprintf(L"-stripmips only applies to cube, volume, array, cubearray, or merge commands\n");
+                    wprintf(L"-stripmips only applies to cube, volume, array, cubearray, merge, or cube-from-* commands\n");
```

Comment:
I don't think ``-stripmips`` does anything for your new functions.

Reply:
Right. so I'll disable another error at line 1501 as well.
like this
```c++
else if ((info.mipLevels > 1) && ((dwOptions & (1 << OPT_STRIP_MIPS)) == 0))
{
    switch (dwCommand)
    {
    case CMD_CUBE_FROM_HC:
    case CMD_CUBE_FROM_VC:
    case CMD_CUBE_FROM_VC_FNZ:
    case CMD_CUBE_FROM_HT:
    case CMD_CUBE_FROM_HS:
    case CMD_CUBE_FROM_VS:
        break;
    default:
        wprintf(L"\nERROR: Can't assemble using input mips. To ignore mips, try again with -stripmips\n");
        return 1;
        break;
   }
}
```